### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by bryanlou**
**Description:** Add unit test for getNamespace function.

### CODE REVIEW

File: exporter/awsemfexporter/metric_translator_test.go [hunk 1]
```diff
@@ -256,6 +256,18 @@ func assertCWMeasurementEqual(t *testing.T, expected, actual CWMeasurement) {
 	assertDimsEqual(t, expected.Dimensions, actual.Dimensions)
 }
 
+func TestGetNamespace(t *testing.T) {
```
Comment (kohrapha) [inline exporter/awsemfexporter/metric_translator_test.go:259]: Can you add the following test cases:
1. `config.Namespace` is non-empty
2. `config.Namespace` and service namespace is also empty
3. Each of the 3 if clauses (i.e. service name and service namespace are strings, only service name is string, only service namespace is string)

**Review by kohrapha — CHANGES_REQUESTED** [review]
Thanks Bryan, just a request for additional test cases to fully test each possible input and edge case.

**Review by kohrapha — APPROVED** [review]
LGTM!
